libs/eo/shared/assets/src/assets/configuration/eo-api-environment.local.json

```
@@ -1,3 +1,3 @@
 {
-  "apiBase": "https://eo-u-<feature-environment>.cloudapp.azure.com/api"
+  "apiBase": "https://eo-u-eo293footer.cloudapp.azure.com/api"
```

issue: Revert to placeholder before merging.
```suggestion
  "apiBase": "https://eo-u-<feature-environment>.cloudapp.azure.com/api"
```